src/openrct2/CMakeLists.txt

```diff
@@ -101,17 +101,9 @@ if (NOT DISABLE_GOOGLE_BENCHMARK)
 endif ()
 
 if (ENABLE_SCRIPTING)
-    if (MSVC)
-        find_package(duktape REQUIRED)
-    else ()
-        PKG_CHECK_MODULES(DUKTAPE REQUIRED duktape)
-        target_include_directories(${PROJECT_NAME} PRIVATE ${DUKTAPE_INCLUDE_DIRS})
-        if (STATIC)
-            target_link_libraries(${PROJECT_NAME} ${DUKTAPE_STATIC_LIBRARIES})
```

Comment:
I'm not sure how the current changes influence making a static build with Duktape. As far as I can tell, I only have shared objects (.so) for Duktape on my machines (macOS/Homebrew and Arch Linux).

Reply:
Should we at least expose the possibility to set `DUKTAPE_STATIC_LIBRARIES` manually for static builds? (Or indeed `LIBRARY`, singular.)

Reply:
I'll leave it up to you – if it's trivial enough to add, let's do this, but don't spend too much on it